lib/assert.js

```diff
@@ -608,6 +620,218 @@ assert.notStrictEqual = function notStrictEqual(actual, expected, message) {
   }
 };
 
+function isPlainObject(obj) {
+  if (typeof obj !== 'object' || obj === null) {
+    return false;
+  }
+  const proto = ObjectGetPrototypeOf(obj);
+  return proto === ObjectPrototype || proto === null || ObjectPrototypeToString(obj) === '[object Object]';
+}
+
+function assertIncludes(actual, expected, loose) {
+  if (expected.length > actual.length) {
+    return false;
+  }
+  if (typeof actual === 'string') {
+    return StringPrototypeIncludes(actual, expected);
+  }
+
+  return expected.every((item) => actual.some((actualItem) => {
+    // eslint-disable-next-line eqeqeq
+    return loose ? actualItem == item : actualItem === item;
+  }));
```

Comment:
I think we should use primordials : `ArrayPrototypeEvery` and `ArrayPrototypeSome`

Reply:
@pmarchini I thought the same but then I looked for the non primordials usage across the codebase and, considering the huge usage of them, I was like "maybe I should not" . Converted tho 😄